Bottom: 8ae79ff40db81d080be8909350ce8c37de55b528
Top:    2ef7290b84f2f295677916d95e93b4ee64305673
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-25 10:19:41 -0500

Update bacula tasks to proper yaml syntax

Several tasks were using parameters to the module instead of children.
Improves readability and keeps tasks up-to-date.



---
diff --git a/roles/bacula-director/tasks/main.yml b/roles/bacula-director/tasks/main.yml
index 7de6cd1..71fa3ce 100644
--- a/roles/bacula-director/tasks/main.yml
+++ b/roles/bacula-director/tasks/main.yml
@@ -5,14 +5,23 @@
     - bacula_monitor_password is defined
     - mariadb_bacula_password is defined
 - name: Install/setup Bacula Director
-  yum: name={{ item }} state=present
+  yum: 
+    name: '{{ item }}'
+    state: present
   with_items:
     - bacula-director
 - name: Assemble dir for backup scripts
-  file: path=/usr/libexec/bacula/backup-scripts state=directory
+  file: 
+    path: /usr/libexec/bacula/backup-scripts
+    state: directory
 - name: Configure Bacula Director
-  template: src=bacula-dir.conf.j2 dest={{ bacula_director_config_dir }}/bacula-dir.conf
-          owner=bacula group=bacula mode=0640 backup=yes
+  template:
+    src: bacula-dir.conf.j2
+    dest: '{{ bacula_director_config_dir }}/bacula-dir.conf'
+    owner: bacula
+    group: bacula
+    mode: 0640
+    backup: yes
   register: bacula_director_config_status
 - name: Use libbaccats-mysql
   command: alternatives --set libbaccats.so /usr/lib64/libbaccats-mysql.so
@@ -25,8 +34,14 @@
 - name: Test Bacula Director configuration
   command: /usr/sbin/bacula-dir -t -c {{ bacula_director_config_dir }}/bacula-dir.conf
 - name: Enable and restart Bacula Director
-  service: name=bacula-dir enabled=yes state=restarted
+  service:
+    name: bacula-dir
+    enabled: yes
+    state: restarted
   when: bacula_director_config_status.changed
 - name: Enable and start Bacula Director
-  service: name=bacula-dir enabled=yes state=started
+  service:
+    name: bacula-dir
+    enabled: yes
+    state: started
   when: not bacula_director_config_status.changed
diff --git a/roles/bacula-storage/tasks/main.yml b/roles/bacula-storage/tasks/main.yml
index 5701b00..01125af 100644
--- a/roles/bacula-storage/tasks/main.yml
+++ b/roles/bacula-storage/tasks/main.yml
@@ -4,20 +4,34 @@
     - bacula_director_password is defined
     - bacula_monitor_password is defined
 - name: Install/setup Bacula Storage
-  yum: name={{ item }} state=present
+  yum: 
+    name: '{{ item }}'
+    state: present
   with_items:
     - bacula-storage
 - name: Configure Bacula Storage
-  template: src=bacula-sd.conf.j2 dest={{ bacula_storage_config_dir }}/bacula-sd.conf
-          owner=bacula group=bacula mode=0640 backup=yes
+  template:
+    src: bacula-sd.conf.j2
+    dest: '{{ bacula_storage_config_dir }}/bacula-sd.conf'
+    owner: bacula
+    group: bacula
+    mode: 0640
+    backup: yes
+
 - name: Check if baculatape selinux module exists
   stat: path=/etc/selinux/targeted/modules/active/modules/baculatape.pp
   register: baculatape_pp
 - import_tasks: selinux-tape-policy.yml
   when: baculatape_pp.stat.exists is defined and baculatape_pp.stat.exists == false
 - name: Make sure bacula user is in the tape group
-  user: name=bacula groups=tape append=yes
+  user: 
+    name: bacula
+    groups: tape
+    append: yes
 - name: Test Bacula Storage configuration
   command: /usr/sbin/bacula-sd -t -c {{ bacula_storage_config_dir }}/bacula-sd.conf
 - name: Enable and start Bacula Storage
-  service: name=bacula-sd enabled=yes state=started
+  service: 
+    name: bacula-sd
+    enabled: yes
+    state: started
diff --git a/roles/bacula-storage/tasks/selinux-tape-policy.yml b/roles/bacula-storage/tasks/selinux-tape-policy.yml
index 7fb6800..c15db31 100644
--- a/roles/bacula-storage/tasks/selinux-tape-policy.yml
+++ b/roles/bacula-storage/tasks/selinux-tape-policy.yml
@@ -2,9 +2,12 @@
   command: mktemp -d /tmp/ansible-bacula-storage.XXXXXXXX
   register: bacula_storage_tmp_dir
 - name: Copy baculatape type enforcement file
-  copy: src=baculatape.te
-          dest={{ bacula_storage_tmp_dir.stdout }}/baculatape.te
-          owner=root group=root mode=0640
+  copy: 
+    src: baculatape.te
+    dest: '{{ bacula_storage_tmp_dir.stdout }}/baculatape.te'
+    owner: root
+    group: root
+    mode: 0640
 - name: Run checkmodule on baculatape.te
   command: /usr/bin/checkmodule -M -m -o baculatape.mod baculatape.te
   args:
@@ -18,4 +21,6 @@
   args:
     chdir: "{{ bacula_storage_tmp_dir.stdout }}"
 - name: Delete temp directory
-  file: path={{ bacula_storage_tmp_dir.stdout }} state=absent
+  file:
+    path: '{{ bacula_storage_tmp_dir.stdout }} '
+    state: absent
